### DISCUSSION TITLE
using simdjson in java language

### DISCUSSION THREAD

**Discussion by wanweiqiangintel** [simdjson/simdjson#1823] [General]
Simdjson is very popular and has been implemented in various languages, such as C++/Rust/Go, but i can't find the implementation in java language. I try to use java foreign linker API([https://openjdk.java.net/jeps/412]) calling simdjson to accelerate json_extract() in Presto([https://github.com/prestodb/presto]), which can get 3~5X speedup. So maybe we can do something(JNI/JavaCPP/Foreign linker api) to make simdjson more popular in the Java world.

**Comment by lemire** [comment]
Duplicate of https://github.com/simdjson/simdjson/issues/121
